packages/vite/src/node/build.ts

```diff
@@ -206,7 +206,7 @@ export interface BuildEnvironmentOptions {
    */
   copyPublicDir?: boolean
   /**
-   * Whether to emit a .vite/manifest.json under assets dir to map hash-less filenames
+   * Whether to emit a .vite/manifest.json to map hash-less filenames
```

Comment:
The other docs hint at the previous options implying the output dir a bit more. Maybe we could be explicit here: 
```suggestion
   * Whether to emit a .vite/manifest.json in the output dir to map hash-less filenames
```